org.freedesktop.Sdk.Extension.rust-stable.json

```diff
@@ -287,6 +287,35 @@
                 "install -m 755 ./target/release/cargo-vendor-filterer /usr/lib/sdk/rust-stable/bin/cargo-vendor-filterer"
             ]
         },
+        {
+            "name": "rust-bindgen",
+            "buildsystem": "simple",
+            "build-options": {
+                "append-path": "/usr/lib/sdk/rust-stable/bin",
+                "env": {
+                    "CARGO_HOME": "/run/build/rust-bindgen/cargo"
+                }
+            },
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/rust-lang/rust-bindgen/",
+                    "commit": "af7fd38d5e80514406fb6a8bba2d407d252c30b9",
+                    "tag": "v0.71.1",
+                    "x-checker-data": {
```

Comment:
Remove the x-checker here. PRs here are automerged and this depends on generated sources. So the new version update PRs will fail as it depends on regenerated sources blocking rust updates.

Reply:
The other option is to opt out of the global x-checker instance and use a custom workflow with auto merge that updates the generated manifest. 

If you want to set up that let me know so that I can enable automerge.

Reply:
> The other option is to opt out of the global x-checker instance and use a custom workflow with auto merge that updates the generated manifest.
> 
> If you want to set up that let me know so that I can enable automerge.

Seems a bit annoying to pull off. I'll just go with removing the x-checker, rust-bindgen doesn't update frequently in the first place.